### ISSUE TITLE
Crashes if gfx/gff/map/sfx/music sections are not present in p8 file

### ISSUE THREAD

**Issue by gingerbeardman** [picolove/picolove#76]
This is the type of file you get from pico8 if you do not create any graphics. At least in the latest version. 

Perhaps older pico8 saved blank sections for those?

**Comment by Shoozza** [comment]
Tested with PICO-8 0.1.8 (version 8 of the cart file format) - it includes prefilled sections (with 0's).  
Later versions don't store those - if empty.  

Here the blank version for PICO-8 0.1.11g:
```
pico-8 cartridge // http://www.pico-8.com
version 16
__lua__
```
We are currently at version 8 of the cart file for which these sections should always exist.

**Comment by gingerbeardman** [comment]
OK, no problem, we should state somewhere that we are compatible with that old version of PICO-8

**Comment by Shoozza** [comment]
Will be fixed as part of #77
